ArduPlane/commands_logic.cpp

```diff
@@ -287,6 +287,7 @@ bool Plane::verify_command(const AP_Mission::Mission_Command& cmd)        // Ret
     case MAV_CMD_DO_AUTOTUNE_ENABLE:
     case MAV_CMD_DO_CONTROL_VIDEO:
     case MAV_CMD_DO_SET_CAM_TRIGG_DIST:
+    case MAV_CMD_OBLIQUE_SURVEY:
```

Comment:
These should all be up in AP_Mission.cpp rather than in each vehicle

Reply:
I followed MAV_CMD_DO_SET_CAM_TRIGG_DIST's implementation, should I just remove it from here?